lively-lions/README.md

```diff
@@ -1,2 +1,76 @@
 # Lively Lions
-This is the default README of your team's project. Please replace this by a README with more information on your project. At the very least, your README should contain information on how to set-up and run your project.
+ Text Based Game (PvP in ROOM)
+
```

Comment:
Well made README, with setup for dev and for running. 

clear and easy to follow.